oasislmf/pytools/gul/manager.py

```diff
@@ -218,21 +225,81 @@ def run(run_dir, ignore_file_type, sample_size, loss_threshold, alloc_rule, debu
         # create the array to store the seeds
         seeds = np.zeros(len(np.unique(items['group_id'])), dtype=Item.dtype['group_id'])
 
+        file_path = os.path.join(input_path, 'correlations.bin')
```

Comment:
yes, this probably fails if there is no `correlations.bin` file.
the MDK should now produce a `correlations.bin` file in any case.
However, I can encapsulate this in a `try..catch so if the file is not there (e.g., for some reason `gulpy` is run in an old losses directory where correlations.bin is absent) it sets `ignore_correlation=True`, prints a log.info() and doesn't error.

Reply:
I've added a try catch block and now it silently continues the computation even if correlations.bin doesn't exist. 
However, this might be risky because, say a user defines correlations in the model settings, but the correlations.bin doesn't get created or for some reason is not updated, the user sees that gulpy runs but doesn't use correlations.

Reply:
So, we need to take a UX decision whether we prefer to break and signal that there's a problem or keep going flagging the issue in the log, with the risk that the user doesn't realise that correlations are not used.
Ideas?